tutorials/ES/01_tour_hub_de_huggingface.md

```diff
@@ -0,0 +1,223 @@
+
+```python
+git lfs track "*.your_extension"
+```
+
+```python
+git lfs install
+```
+
+d. Agregue sus archivos al repositorio. Los archivos dependen del framework/bibliotecas que esté utilizando. En general, lo importante es que proporcione todos los artefactos necesarios para cargar el modelo. Por ejemplo:
+
+1. Para TensorFlow, es posible que desee cargar un archivo de modelo guardado o `h5`.
+2. Para PyTorch, por lo general, es un `pytorch_model.bin`.
+3. Para Scikit-Learn, generalmente es un archivo `joblib`
+
+Aquí hay un ejemplo en Python guardando un archivo de modelo de Scikit-Learn.
+
+```python
+from sklearn import linear_model
+reg = linear_model.LinearRegression()
+reg.fit([[0, 0], [1, 1], [2, 2]], [0, 1, 2])
+
+from joblib import dump, load
+dump(reg, 'model.joblib')
+```
+
+e. Commit y push a sus archivos (asegúrese de que el archivo guardado esté dentro del repositorio).
+
+```python
+git add .
+git commit -m "First model version"
+git push
+```
+
+¡Y hemos terminado! ¡Puede consultar su repositorio con todos los archivos agregados!
+
+![Untitled](../images/Untitled%205.png)
+
+La interfaz de usuario le permite explorar los archivos del modelo y los commits. Puede ver la diferencia introducida por cada commit.
+
+**Reto 4**. ¡Te toca a ti! Cargue un modelo ficticio de la biblioteca de su elección.
+
+Ahora que el modelo está en el Hub, ¡otros pueden encontrarlo! También puede colaborar con otros fácilmente creando una organización. El alojamiento a través del Hub permite que un equipo actualice los repositorios y haga cosas a las que podría estar acostumbrado, como trabajar en branches y en colaboración. El Hub también permite la creación de versiones en sus modelos: si un punto de control (checkpoint) del modelo se rompe, siempre puede volver a una versión anterior.
+
+En la parte superior de `README` puede encontrar algunos metadatos. Ahora mismo solo encontrarás la licencia pero puedes añadir más cosas. Probemos algo de esto:
+
+```yaml
+ tags:
+- es       # Esto se detectará automáticamente como una etiqueta de idioma.
+- bert     # Puede tener etiquetas adicionales para filtrar
+- text-classification # Se detectará automáticamente como una etiqueta de tarea.
+datasets:
+- llamas # Esto se vinculará a un conjunto de datos en el Hub, si existe.
+```
+
+**Reto 5**. Utilizando la [documentación](https://huggingface.co/docs/hub/model-repos#how-are-model-tags-determined) cambie el ejemplo predeterminado en el widget.
+
+Los metadatos permiten que las personas descubran su modelo rápidamente. Tu modelo ahora aparecerá cuando busques modelos de clasificación de texto en español. El modelo también aparecerá al mirar el conjunto de datos.
+
+Espera... ¿conjuntos de datos?
```

Comment:
dataset is widely used by now in Spanish as well, so I would just say dataset in all of these, that way you don't go from one to the other